conformance/utils/suite/experimental_reports.go

```diff
@@ -183,3 +185,25 @@ func isTestExtended(profile ConformanceProfile, test ConformanceTest) bool {
 	}
 	return false
 }
+
+// buildSummary creates a human-readable message about each profile's test outcomes.
+func buildSummary(report confv1a1.ProfileReport) string {
+	reportMessage := fmt.Sprintf("Core tests %s", buildReportMessage(report.Core))
+	if report.Extended != nil {
+		reportMessage = fmt.Sprintf("%s. Extended tests %s", reportMessage, buildReportMessage(report.Extended.Status))
+	}
+	return fmt.Sprintf("%s.", reportMessage)
+}
```

Comment:
nit, just a suggestion for an alternative:

```suggestion
func buildSummary(report confv1a1.ProfileReport) (reportMessage string) {
	if report.Extended == nil {
		reportMessage = fmt.Sprintf("Core tests %s", buildReportMessage(report.Core))
	} else {
		reportMessage = fmt.Sprintf("%s. Extended tests %s", reportMessage, buildReportMessage(report.Extended.Status))
	}
	return
}
```

Reply:
It is not the same, as the core results need to be always included in the summary. I've done a small change to the function by following the approach you suggested though: b91df55766cefd366758ca445daa68e187e1209f